integreat_compass/cms/templates/interactions/vote_form.html

```diff
@@ -0,0 +1,67 @@
+        {% for offer_version in pending_offer_versions %}
+            <form action="{% url 'cms:protected:add_vote' offer_version.id %}"
+                  method="post">
+                {% csrf_token %}
+                <div class="flex flex-row border-2 border-gray-200 rounded-lg shadow-md shadow-inner p-6 mb-8">
+                    <div class="basis-2/3 pr-8">
+                        <p class="text-xl font-bold">{{ offer_version.title }}</p>
+                        <p>{{ offer_version.offer.organization.org_name }}</p>
+                        <p class="text-lg text-gray-700 text-bold pt-4">{% translate "short description" %}</p>
+                        <p class="text-gray-600">{{ offer_version.description }}</p>
+                        <div class=" my-4">
+                            {% for tag in offer_version.offer.tags.all %}
+                                <span class="bg-red-500 text-white rounded-full py-1 px-2">{{ tag }}</span>
+                            {% endfor %}
+                        </div>
+                        <p class="text-gray-700">{% translate "created at" %}: {{ offer_version.created_at }}</p>
+                    </div>
+                    <div class="basis-1/3 flex flex-col justify-around align-center font-bold">
+                        {% if offer_version.user_vote %}
+                            {% if offer_version.user_vote.approval %}
+                                <p class="text-green-500">{% translate "You have approved this application" %}</p>
+                            {% else %}
+                                <p class="text-red-500">{% translate "You have declined this application" %}</p>
+                            {% endif %}
+                        {% else %}
+                            <p class="text-gray-700">{% translate "Your vote is still pending" %}</p>
+                        {% endif %}
+                        <div class="grid grid-cols-3 gap-8 font-bold">
+                            <div>
+                                <p>{% translate "approves" %}</p>
+                                <p class="text-green-600">{{ offer_version.approves }}/{{ number_of_board_members }}</p>
+                            </div>
+                            <div>
+                                <p>{% translate "declines" %}</p>
+                                <p class="text-red-600">{{ offer_version.declines }}/{{ number_of_board_members }}</p>
+                            </div>
+                            <div>
+                                <p>{% translate "pending" %}</p>
+                                <p>{{ offer_version.pending }}/{{ number_of_board_members }}</p>
+                            </div>
+                        </div>
+                        <div class="grid grid-cols-3 gap-8 text-white ">
+                            <button class="bg-blue-500 hover:bg-blue-700 rounded-full" type="button">{% translate "to homepage" %}</button>
```

Comment:
```suggestion
                            <a href="{{ organization.web_address }}" class="bg-blue-500 hover:bg-blue-700 rounded-full" type="button">{% translate "to homepage" %}</button>
```
Since I understand this button as a link to the website or Social Media of the organization, that just looks like a button, I would change it to a anker-tag. We store the web_address our model Organization. If you decide to follow this suggestion, you would still need to add the organization to the according view

Reply:
Hmh, I thought this will link to to detail-page of this offer :thinking:

Reply:
Ah okay, I understand. I misinterpreted the design then. I think then I find "to website" somewhat misleading, maybe "see more details" would be more intuitive?

Reply:
> Hmh, I thought this will link to to detail-page of this offer thinking

Are we going to do this as a modal, like when clicking on an offer in the Filter-view? While it would make sense, a modal on a modal isn't great. On the other hand, openingn the details in its own page would require a new (and so-far unplanned) view

Reply:
Yes I thought we would open (reuse) the modal. What would be the other modal? Don't see why this leads to a modal on a modal situation.

Reply:
Oh sorry, nevermind, I thought this was openeed through the rejection-comment-modal 🤦🏼‍♀️

Reply:
```suggestion
                            <button class="bg-blue-500 hover:bg-blue-700 rounded-full" type="button">{% translate "Details" %}</button>
```
I agree with @JoeyStk that "to homepage" is not a good label if it does not lead to the homepage.